src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/ManualComparisonService.kt

```diff
@@ -60,7 +61,17 @@ class ManualComparisonService(
     val nomisDates = objectMapper.convertValue(comparisonPerson.nomisDates, object : TypeReference<Map<ReleaseDateType, LocalDate?>>() {})
     val overrideDates = objectMapper.convertValue(comparisonPerson.overrideDates, object : TypeReference<Map<ReleaseDateType, LocalDate?>>() {})
     val breakdownByReleaseDateType = objectMapper.convertValue(comparisonPerson.breakdownByReleaseDateType, object : TypeReference<Map<ReleaseDateType, ReleaseDateCalculationBreakdown>>() {})
-    return transform(comparisonPerson, nomisDates, calculatedReleaseDates, overrideDates, breakdownByReleaseDateType)
+    val sdsSentences = if (comparisonPerson.sdsPlusSentencesIdentified.isEmpty) emptyList<SentenceAndOffences>() else objectMapper.convertValue(comparisonPerson.sdsPlusSentencesIdentified, object : TypeReference<List<SentenceAndOffences>>() {})
+    val sdsCaseAndCount = buildSdsCaseAndCount(sdsSentences)
+    return transform(comparisonPerson, nomisDates, calculatedReleaseDates, overrideDates, breakdownByReleaseDateType, sdsCaseAndCount)
+  }
+
+  private fun buildSdsCaseAndCount(sentencesAndOffences: List<SentenceAndOffences>): List<String> {
+    val caseAndCountList = mutableListOf<String>()
+    sentencesAndOffences.forEach {
+      caseAndCountList.add("Court case ${it.caseSequence}, count ${it.lineSequence}")
```

Comment:
This does embed some display into the api. Happy to go with it now, but if we have a future content change, we should return objects to make it easier to customise.